tools/keytools/keygen.c

```diff
@@ -0,0 +1,351 @@
+/* keygen.c
```

Comment:
Found several unused var warnings. Also the Makefile in tools/keytool is not updated. Pushed patch here: https://github.com/dgarske/wolfBoot/commit/40ddabe6bccb4ce67bb6cb8877e89afec8cae85f
Having trouble pushing into your IAR branch.

Reply:
branch is 'iar'

Reply:
cherry-picked 40ddabe - fixed